<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addOrderDialog</class>
 <widget class="QDialog" name="addOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Instr Info</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="underlyingInstrTypeLabel">
            <property name="text">
             <string>Underlying Instr Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="strikePriceLabel">
            <property name="text">
             <string>Strike Price</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="strikePriceLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="maturityLabel">
            <property name="text">
             <string>Maturity</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="callPutLabel">
            <property name="text">
             <string>Call/Put</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="callPutComboBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="optionTypeLabel">
            <property name="text">
             <string>Option Type</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="optionTypeComboBox"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="underlyingInstrTypeComboBox"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="underlyingCodeLabel">
            <property name="text">
             <string>Underlying Code</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="underlyingCodeLineEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="underlyingPriceLabel">
            <property name="text">
             <string>Underlying Price</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="underlyingPriceLineEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="knockOutPriceLabel">
            <property name="text">
             <string>Knock-Out Price</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="knockOutPriceLineEdit"/>
          </item>
          <item row="2" column="1">
           <widget class="QDateEdit" name="maturityDateEdit">
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="volTypeLabel">
            <property name="text">
             <string>Vol Type</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="volTypeComboBox"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="impliedVolLabel">
            <property name="text">
             <string>Implied Vol</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="impliedVolLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Transaction Info</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="priceLabel">
            <property name="text">
             <string>Price</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="priceLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="amountLabel">
            <property name="text">
             <string>Amount</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="amountLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="clientIDLabel">
            <property name="text">
             <string>Client ID</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="clientIDLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="longShortLabel">
            <property name="text">
             <string>Long/Short</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="longShortComboBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addOrderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addOrderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
